internal/mode/static/telemetry/collector_test.go

```diff
@@ -379,14 +405,43 @@ var _ = Describe("Collector", Ordered, func() {
 					RouteAttachedClientSettingsPolicyCount:   2,
 					ObservabilityPolicyCount:                 1,
 					NginxProxyCount:                          1,
+					SnippetsFilterCount:                      3,
 				}
 				expData.ClusterVersion = "1.29.2"
 				expData.ClusterPlatform = "kind"
 
-				data, err := dataCollector.Collect(ctx)
+				expData.SnippetsFiltersContextDirectives = []string{
+					"http-aio",
+					"location-keepalive_time",
+					"main-worker_priority",
+					"server-auth_delay",
+					"http-client_body_timeout",
+					"location-allow",
+					"main-worker_rlimit_nofile",
+					"server-ignore_invalid_headers",
+				}
+				expData.SnippetsFiltersContextDirectivesCount = []int64{
+					2,
```

Comment:
Can we add one directive that has 3 values to ensure it's at the top of the list?